src/core/GetLiveChatConfigOptionalParams.ts

```diff
@@ -1,6 +1,7 @@
 interface GetLiveChatConfigOptionalParams {
     sendCacheHeaders?: boolean;
     useRuntimeCache?: boolean;
+    useParallelLoad?: boolean;

```

Comment:
Would it make more sense if  `useParallelLoad` was  part of `InitializeOptionalParams` instead?

GetLiveChatConfigOptionalParams is the params used specifically for getting live chat config. 

`sendCacheHeaders` is applied upon getting live chat config call
`useRuntimeCache` is applied to retrieve the chat config from the `ChatSDK` instance 

`useParallelLoad` would make more sense if it in the `initialization` context since it's not for getting live chat config specifically.

Reply:
Would it make sense to use `useConcurrency` instead? `Promise.all()` technically execute the promises concurrently and not parallelly

Reply:
![image](https://github.com/user-attachments/assets/5ef4a35a-628d-43d0-9687-7aff39f367e1)